src/components/SsnInput/SsnInput.js

```diff
@@ -63,32 +62,30 @@ class SsnInput extends React.Component {
   componentDidMount = () => {
     const inputElement = this.inputRef.current
 
-    bindTextInputUiEvents(inputElement)
-
     // Add event listener if this component is uncontrolled
     if (this.props.value === '') {
       inputElement.addEventListener('input', this.handleInput)
     } else {
       this.maskValue()
     }
-  }
+  };
 
   componentDidUpdate = () => {
     const { value } = this.props
 
     // Mask value if this component is controlled
     if (value !== '' && value !== this.maskedValue) this.maskValue()
-  }
+  };
 
   handleChange = event => {
     event.target.value = event.target.value.replace(/-/g, '')
-    this.props.onChange(event)
-  }
+    this.props.onChange ? this.props.onChange(event) : null
```

Comment:
wat?

Reply:
it's checking that there is an on change function